src/components/InputPhoneForm/InputPhoneForm.css

```diff
@@ -0,0 +1,31 @@
+.phone-input-container {
+  position: relative;
+}
+
+.phone-input-select {
+  position: absolute;
+  left: .5rem;
+  top: 50%;
+  transform: translateY(-50%);
+  padding: .5rem .25rem;
+  color: var(--clr-background);
+  border: 2px solid var(--clr-secondary-60);
+  border-radius: var(--border-sm);
+  min-width: 4.5rem;
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  background: white
+    url('data:image/svg+xml;charset=US-ASCII,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 4 5"><path fill="none" stroke="white" d="M0 0l2 2 2-2"/></svg>')
+    no-repeat right 16px center;
+  background-color: var(--clr-primary);
+  background-size: 12px;
+}
+
+.phone-input-number {
+  padding: 1rem;
+  width: 100%;
+  border: 2px solid var(--clr-secondary-60);
+  border-radius: var(--border-sm);
+  text-align: right;
```

Comment:
It looks much better if you align it to the left.

Reply:
Ready, change tex-align to left.